test/unit/data/attachments.js

```diff
@@ -12,7 +12,9 @@ describe('.zip attachments streaming', () => {
       { row: { instanceId: 'subone', name: 'secondfile.ext', content: 'this is my second file' } },
       { row: { instanceId: 'subtwo', name: 'thirdfile.ext', content: 'this is my third file' } }
     ]);
-    zipStreamToFiles(zipStreamFromParts(streamAttachments(inStream)), (result) => {
+    zipStreamToFiles(zipStreamFromParts(streamAttachments(inStream)), (err, result) => {
+      if(err) return done(err);
```

Comment:
We don't lint test files at the moment, but I think the linter requires a space between `if` and `(`. I don't think that should block merging, but maybe that's all the more reason to try to get #545 merged soon.